pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/BrokersBase.java

```diff
@@ -96,21 +97,24 @@
     @GET
     @Path("/leaderBroker")
     @ApiOperation(
-            value = "Get the service url of the leader broker in the cluster.",
-            response = String.class)
+            value = "Get the information of the leader broker.",
+            response = BrokerInfo.class)
     @ApiResponses(
             value = {
                     @ApiResponse(code = 401, message = "Authentication required"),
-                    @ApiResponse(code = 403, message = "This operation requires super-user access") })
-    public String getLeaderBroker() throws Exception {
+                    @ApiResponse(code = 403, message = "This operation requires super-user access"),
+                    @ApiResponse(code = 404, message = "Leader broker not found") })
+    public BrokerInfo getLeaderBroker() throws Exception {
         validateSuperUserAccess();
 
         try {
-            return pulsar().getLeaderElectionService().getCurrentLeader()
-                    .map(LeaderBroker::getServiceUrl)
-                    .orElse(null);
+            LeaderBroker leaderBroker = pulsar().getLeaderElectionService().getCurrentLeader()
+                    .orElseThrow(() -> new RestException(Status.NOT_FOUND, "Couldn't find leader broker"));
+            BrokerInfo brokerInfo = new BrokerInfo();
```

Comment:
I create BrokerInfo which is the same as LeaderBroker, Because I can not access LeaderBroker in module pulsar-client-admin.

Reply:
@eolivelli  Please help to review this PR.